.gitignore

```diff
@@ -41,4 +46,25 @@ tests/data/*_test.*
 
 # Generated files
 *_export.h
-/.vscode
\ No newline at end of file
+/.vscode
+cmake_install.cmake
+CMakeCache.txt
+CTestTestfile.cmake
+Makefile
+tests/CMakeLists.txt
+tests/data/feng_r.txt
+tests/data/feng.txt
+tests/data/lid/*.out
+tests/data/lid/*.rpt
+tests/data/lid/revised/*.out
+tests/data/lid/revised/*.rpt
+tools/swmm-output/CMakeFiles/*
+tools/swmm-output/CMakeFiles/swmm-output.dir/build.make
+tools/swmm-output/CMakeFiles/swmm-output.dir/C.includecache
+tools/swmm-output/CMakeFiles/swmm-output.dir/cmake_clean.cmake
+tools/swmm-output/CMakeFiles/swmm-output.dir/depend.internal
+tools/swmm-output/CMakeFiles/swmm-output.dir/depend.make
+tools/swmm-output/CMakeFiles/swmm-output.dir/DependInfo.cmake
+tools/swmm-output/CMakeFiles/swmm-output.dir/flags.make
+tools/swmm-output/CMakeFiles/swmm-output.dir/link.txt
+tools/swmm-output/CMakeFiles/swmm-output.dir/progress.make
```

Comment:
Why ignoring all of this?
Surely the CMakeLists.txt ought to be tracked by git, aren't they?

Reply:
@lrntct Response to you comments from July 4, 2020:

 - The header files were moved to include in order to test some library options but I agree with you, they shouldn't be there and moved them back to src.
 
- You are right, the .out and .rpt files are not needed. I removed them.

- All those files, except tests/CMakeLists.txt, are generated every time in running CMake, or are SWMM generated .out and .rpt files. Please note that are not in the original repository either. I agree with you that tests/CMakeLists.txt is needed, and re-established it in subsequent commits, along with other changes.